ckanext/canada/i18n/fr/LC_MESSAGES/ckanext-canada.po

```diff
@@ -4882,3 +5161,268 @@ msgstr "Mémoriser mes informations"
+#~ "can be entered in this field.\n"
+#~ msgstr ""
+#~ "Si la valeur XX (aucun) est "
+#~ "inscrite, alors aucune autre valeur ne"
+#~ " peut être inscrite dans ce champ."
+
+#~ msgid ""
+#~ "This field must be N, No or Non, if the Procurement Strategy for\n"
+#~ "Aboriginal Business field is MS or VS.\n"
+#~ msgstr ""
+#~ "Ce champ doit être N, No ou "
+#~ "Non, si le champ Stratégie "
+#~ "d’approvisionnement auprès des entreprises "
+#~ "autochtones contient MS ou VS."
+
+#~ msgid ""
+#~ "This field must be populated with "
+#~ "a 1 if the solicitation procedure "
+#~ "is\n"
+#~ "identified as non-competitive (TN) or"
+#~ " Advance Contract Award Notice (AC).\n\n"
+#~ msgstr ""
+#~ "Cette zone doit contenir un « 1"
+#~ " » si la méthode d’invitation à "
+#~ "soumissionner est identifiée comme étant "
+#~ "non concurrentielle (TN) ou un préavis"
+#~ " d’attribution de contrat (AC)."
+
+#~ msgid "Must be left blank when trade agreements specified\n"
+#~ msgstr ""
+
+#~ msgid ""
+#~ "If the value NA (not applicable) is entered, then no\n"
+#~ "other value can be entered in this field.\n"
+#~ msgstr ""
+#~ "Si la valeur NA (sans objet) est"
+#~ " inscrite, alors aucune autre valeur "
+#~ "ne peut être inscrite dans ce "
+#~ "champ."
+
+#~ msgid ""
+#~ "If TC, TN or AC is selected in the Solicitation Procedure data field\n"
+#~ "with a value other than XX (None)"
+#~ " selected in the Trade Agreement data"
+#~ "\n"
+#~ "field, then a Limited Tendering value"
+#~ " other than 00 (none) must be "
+#~ "entered.\n"
+#~ msgstr ""
+#~ "Si TC, TN ou AC est sélectionné"
+#~ " dans le champ de données Méthode "
+#~ "d’invitation à soumissioner avec une "
+#~ "valeur autre que XX (Aucune) "
+#~ "sélectionnée dans le champ de données"
+#~ " Accord commercial, une valeur d'offre "
+#~ "limitée autre que 0 (aucune) doit "
+#~ "être entrée."
+
+#~ msgid ""
+#~ "If this field is populated, it must be with a “0” if the procurement\n"
+#~ "was identified as non-competitive (TN)"
+#~ " or advance contract award notice\n"
+#~ "(AC) or was identified as an "
+#~ "Amendment (A) in the Instrument type "
+#~ "data field.\n"
+#~ msgstr ""
+#~ "Si cette zone est remplie, il doit"
+#~ " contenir un « 0 » si "
+#~ "l’acquisition a été identifiée comme "
+#~ "étant non concurrentielle (TN) ou un "
+#~ "préavis d’attribution de contrat (AC) ou"
+#~ " si l’acquisition a été identifiée "
+#~ "comme un modification (A) dans le "
+#~ "champ Type d’instrument."
+
+#~ msgid "This field is limited to only 3 or 4 digits.\n"
+#~ msgstr "Ce champ est limité à 3 ou 4 caractères."
+
+#~ msgid ""
+#~ "This field must contain the first three digits of a postal\n"
+#~ "code in A1A format or the value \"NA\"\n"
+#~ msgstr ""
+#~ "Ce champ doit contenir les trois "
+#~ "premiers chiffres d’un code postal dans"
+#~ " le format A1A ou la valeur «"
+#~ " NA. »"
+
+#~ msgid "Please enter a decimal value between zero and one: %"
+#~ msgstr ""
+
```

Comment:
is this where we accumulate old unmatched translations?

Reply:
Okay this seems to be solved now with the Loader class. albeit we have never had a translation for this one, so waiting on that still.